src/components/ChatView.vue

```diff
@@ -78,7 +78,7 @@ export default {
 	data() {
 		return {
 			isDraggingOver: false,
-			isChatScrolledToBottom: getCapabilities()?.spreed?.config?.chat?.legacy || false,
+			isChatScrolledToBottom: getCapabilities()?.spreed?.config?.chat?.legacy || true,
```

Comment:
This basically rolls all the context changes back...

Reply:
We still have a property `this.loadChatInLegacyMode` inside MessageList.vue, which basically does the same check.

I could change it back to `false`, but button is flickering in that case